docs/block-node/block-node-overview.md

```diff
@@ -0,0 +1,137 @@
+- **Data services** - Exposes gRPC/REST APIs providing real-time block streaming, random-access block retrieval, state queries at specific block heights, and cryptographic proofs to Mirror Nodes, other Block Nodes, and applications.
+
+Block Nodes fan out block streams to Mirror Nodes and Tier 2+ Block Nodes while also serving reconnect and state snapshot services back to Consensus Nodes, creating a scalable data availability layer between consensus and downstream consumers.
+
+![block-node-network-architecture](../../assets/block-node-network-architecture.svg)
+
+### How Data Flows Through the Network
+
+The diagram above illustrates the complete data flow:
+
+1. **Consensus Nodes produce blocks** - Users submit transactions via gRPC to Consensus Nodes, which reach consensus through hashgraph and produce finalized blocks with block proofs containing aggregated signatures.
+2. **Block Nodes receive and verify** - Tier 1 Block Nodes receive block streams directly from Consensus Nodes, verify block integrity using aggregated signatures (as defined in [HIP-1200](https://hips.hedera.com/hip/hip-1200)), and store verified blocks and state to local disk and remote archives.
+3. **Block Nodes distribute downstream** - block streams fan out to:
+   - **Mirror Nodes** - for public REST APIs and explorer services
+   - **Tier 2+ Block Nodes** - for geographic redundancy and permissionless participation
+   - **Applications** - via gRPC/REST APIs for custom integrations
+4. **Block Nodes support Consensus Node recovery** - When a Consensus Node experiences downtime or falls behind, it requests reconnect data (recent blocks and state snapshots) from Block Nodes to quickly resynchronize without burdening other Consensus Nodes.
+
+## Key Responsibilities
+
+Block Node operators are responsible for:
+
+- Keeping their Block Node online, monitored, and synchronized with the latest network state.
+- Managing storage and retention for blocks and state snapshots.
+- Securing access to APIs and infrastructure, including authentication, authorization, and network boundaries.
+- Applying software upgrades in line with Hedera/Hiero releases and block node compatibility guidance.
+- Choosing and configuring which services to expose (streaming, random access, proofs, reconnect/state snapshots) for their consumers.
+
+## Benefits for Operators
+
+- Replaces legacy record stream polling with efficient streaming
+- Lower costs via local storage and no S3 operations
+- Enhanced confidence via aggregated network signatures and zero-knowledge proofs.
+- Enhanced Mirror Node integration with random-access and proofs
+- Plugin extensibility without forking
+
+## Choosing Your Block Node Configuration
+
+Use this decision guide to determine which Block Node configuration suits your needs:
+
+### Tier 1 vs Tier 2
+
+**Choose Tier 1 if:**
+
+- You are a Hedera Governing Council member or trusted network partner.
```

Comment:
This is context of a Hedera network. A Hiero network may not have this kind of requirements